### DISCUSSION TITLE
When a corpse brought from the wreckage disappears from the submarine, everything that was in its inventory disappears.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15014] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

This happens at the end of the round and sometimes even during the round.
This is a cruel injustice.
Make everything fall out on the deck (from monsters) or leave a duffel bag after human and husks corpses.

### Reproduction steps

1. Bring a corpse from the wreckage with inventory full with excellent loot
2. leave loot in a corpse inventory
3. Go to next station
4. Dock and oops, loot disappeared as if it had never been there.
5. Cry loser, you just forgot to put the loot in the closet
This is too cruel to be the truth
I already brought all these valuables to my boat. What could go wrong?

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Guh? 
https://github.com/FakeFishGames/Barotrauma/blob/master/Barotrauma/BarotraumaShared/SharedSource/Map/Levels/Level.cs#L4973

Was it a normal corpse or husk playing dead?
I don't remember such problem in 1.5.9.1

**Reply by pavel4win** [reply]
it was normal corpses found in wreck, not husks.
I put loot in they inventory. But forget to get it out midround.
they despawned at the end of round with all items in they inventory.
Anyway midround despawn I turn off with 
corpsedespawndelay="60000"
 corpsespersubdespawnthreshold="500"
commands in my config_player.xml
